packages/webapp/src/util/animal.ts

```diff
@@ -31,3 +33,8 @@ export const generateUniqueAnimalId = (
 ): string => {
   return `${ANIMAL_ID_PREFIX['key' in entity ? 'DEFAULT' : 'CUSTOM']}_${entity.id}`;
 };
+
+export const generateInventoryId = (animalOrBatch: Animal | AnimalBatch): string => {
+  const key = 'count' in animalOrBatch ? AnimalOrBatchKeys.BATCH : AnimalOrBatchKeys.ANIMAL;
```

Comment:
Instead of relying on 'count', I wonder if it might be okay to just pass the key from useAnimalInventory(). 

Also is `internal_identifier` not still passed that we could use?

Reply:
That's what I initially did, and I don't mind changing it back.

We could use `internal_identifier`; I just thought the new key would make the actions easier.
For example, when removing animals and batches, we can just separate animal and batch keys and use their ids.
- animal keys [,,,,] => call remove animals API
- animal batch keys [,,,,] => call remove animal batches API

If selectedIds are `internal_identifier`, we need to find animals and batches from `inventory` again to get animal/batch and their ids. If that's better (it feels more reliable?), we can use `internal_identifier`.

Reply:
Ohh I understand now. It is up to you. We also have `batch` property on the row too: 
![Screenshot 2024-03-14 at 11 42 14 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/8daa4ba8-1430-4c13-b85a-585e0549f5ed)

Reply:
I am not sure how I can use that `batch` effectively...
I will leave it for now!